libkernelflinger/log.c

```diff
@@ -169,5 +187,100 @@ void log(const CHAR16 *fmt, ...)
 
 	va_start(args, fmt);
 	vlog(fmt, args);
+
+	if (KERNELFLINGER_LOG_OUTPUT) {
+		log_file(fmt, args);
+	}
+
 	va_end(args);
 }
+
+#if DEBUG_MESSAGES
+EFI_FILE * log_init(const CHAR16 *filename)
+{
+
+    EFI_FILE *root;
+	EFI_FILE *file;
+    EFI_STATUS ret;
+
+	root = g_root;
+
+	if (!root)
+		return NULL;
+
+	file = NULL;
```

Comment:
file should be initialized when declared.

Reply:
Already modified.